server/data/prisonerLocationApiClient.ts

```diff
@@ -13,20 +14,20 @@ export interface Downloads {
 }
 
 export default class PrisonerLocationApiClient extends RestClient {
-  constructor() {
-    super('Prisoner Location Api Client', config.apis.prisonerLocationApi as ApiConfig, logger)
+  constructor(authenticationClient: AuthenticationClient) {
+    super('Prisoner Location Api Client', config.apis.prisonerLocationApi as ApiConfig, logger, authenticationClient)
   }
 
-  todaysFile(token: string): Promise<Download> {
-    return this.get<Download>({ path: '/today', errorHandler: this.handleNotFoundError }, asUser(token))
+  todaysFile(username: string): Promise<Download> {
+    return this.get<Download>({ path: '/today', errorHandler: this.handleNotFoundError }, asSystem(username))
```

Comment:
I'd switched this to `asUser` previously, but hadn't realised that there was middleware that was converting the user token into a system token anyway